cmd/ocm-backplane/cloud/common.go

```diff
@@ -274,7 +275,17 @@ func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials
 		)
 
 	}
-	initialClient, err := StsClient(cfg.BackplaneConfiguration.ProxyURL)
+	// Use AWS-specific proxy for initial STS client, fallback to regular proxy
+	// Priority: 1) BACKPLANE_AWS_PROXY env var; 2) regular proxy from local backplane config
+	var stsProxyURL *string
+	if awsProxy := os.Getenv("BACKPLANE_AWS_PROXY"); awsProxy != "" {
```

Comment:
What if moving awsProxy to backplane-config struct and fallback to proxy if not defined? 
https://github.com/openshift/backplane-cli/blob/main/pkg/cli/config/config.go#L37 

Then you may be able to use it as `cfg.BackplaneConfiguration.AwsProxy`




Pls move `BACKPLANE_AWS_PROXY` env vars : https://github.com/openshift/backplane-cli/blob/main/pkg/info/info.go#L12

Reply:
I was planning to implement a flag-based option to specify the AWS proxy at runtime (instead of relying on an environment variable or the local backplane config). However, since we’re approaching the end of the sprint, I’m considering deferring this to a later implementation. WDYT?

Reply:
Before implementing the flag-based approach, it could be a good idea to resolve this repetition within the `backplane-config` struct.